src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -342,7 +342,7 @@ private void updateTransactionTableView() {
             this.transactionTableView.setItems(observableTransactionModels);
 
         } catch (StorageException e) {
-            System.out.println("Error of loading all transactions into list view: " + e.getMessage());
+            setupErrorPopup("Error loading all transactions into list view.");
```

Comment:
Should probably keep the message. Latter definitely want to translate it but even the bare message should be useful.